src/render/patterns.js

```diff
@@ -0,0 +1,54 @@
+import * as renderUtils from './utils';
+import * as utils from '../utils';
+
+/**
+ * For any given `patterns` entry, render a pattern-collection page for
+ * its `items`. Also, remove the `contents` property for individual patterns.
+ * Patterns will not render individual pages. This function mutates `patterns`.
+ *
+ * @param {Object} patterns      The current level of the patterns tree we're
+ *                               rendering
+ * @param {Object} drizzleData   All the data we have, including `options`
+ * @param {String} collectionKey The key of the current set of `patterns`.
+ *                               Used to derive the collection's "name"
+ */
+function renderPatternCollection (patterns, drizzleData, collectionKey) {
+  for (var individualPatternKey in patterns.items) {
+    // We want to render only pattern-collection pages, not individual
+    // patterns. Remove `contents` from individual patterns so the write
+    // phase doesn't create files for them.
+    delete patterns.items[individualPatternKey].contents;
+  }
+  patterns.name = utils.titleCase(utils.keyname(collectionKey));
+  patterns.contents = renderUtils.applyTemplate(
+    drizzleData.layouts.patternCollection.contents, // TODO obviously fragile
```

Comment:
I'm referencing a deep-nested object without testing to make sure that it's defined. It's temporary until I clean up how I want to handle identifying the right template for pattern collections.